autoload/shiny.vim

```diff
@@ -5,20 +5,28 @@ set cpo&vim
 let s:V = vital#shiny#of()
 let s:Highlight = s:V.import('Coaster.Highlight')
 
+let s:vim_shiny_hi_paste = get(g:, 'vim_shiny_hi_paste', 'Shiny')
+
+function! s:initialize()
```

Comment:
**[vint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Use the abort attribute for functions in autoload (see Google VimScript Style Guide (Functions))